experimenter/experimenter/experiments/tests/api/v6/test_serializers.py

```diff
@@ -403,3 +405,26 @@ def test_serializer_outputs_empty_targeting(self):
         serializer = NimbusExperimentSerializer(experiment)
         self.assertEqual(serializer.data["targeting"], "true")
         check_schema("experiments/NimbusExperiment", serializer.data)
+
+    def test_localized_desktop(self):
+        locale_en_us = LocaleFactory.create(code="en-US")
+        locale_en_ca = LocaleFactory.create(code="en-CA")
+        locale_fr = LocaleFactory.create(code="fr")
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.ENDING_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.DESKTOP,
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_113,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.NO_TARGETING,
+            channel=NimbusExperiment.Channel.NIGHTLY,
+            primary_outcomes=["foo", "bar", "baz"],
+            secondary_outcomes=["qux", "quux"],
+            is_localized=True,
+            locales=[locale_en_us, locale_en_ca, locale_fr],
+        )
+
+        serializer = NimbusExperimentSerializer(experiment)
+
```

Comment:
We should also use `check_schema` here to make sure it passes the shared schema validator.